JASP-Tests/R/tests/testthat/test-regressionlogistic.R

```diff
@@ -299,4 +299,44 @@ test_that("Error Handling", {
   results <- jasptools::run("RegressionLogistic", "debug.csv", options)
   status <- results[["status"]]
   expect_identical(status, "validationError")
-})
\ No newline at end of file
+})
+
+test_that("Pseudo R-squared are correct", {
+  # Specifically, tests McFadden, Nagelkerke, Tjur, and Cox & Snell coefficients
+  # Example from Tjur (2008) http://dx.doi.org/10.1198/tast.2009.08210, page 370
+  # Tjur's results for Tjur R2: 0.096231
+  #.
+  # Check against code from performance package
+  # library(aplore3)
```

Comment:
Great! A minor comment, could you add the package versions as a comment? If they ever change and mess up then we can retrace the package versions where it did work.